File: dados/estatistica3/poisson.py
from scipy.stats import poisson

# média de 2 acidentes
# qual a probabilidade de ocorrer 3 ?

poisson.pmf(3, 2)

# e a probabilidade de serem 3 ou menos acidentes ?

poisson.cdf(3, 2)

# e mais de 3 acidentes ?

poisson.sf(3, 2)

# Em média ocorrem 10 mortes por mês em um hospital.

# Qual a probabilidade de, em um determinado mês, ocorrerem 12 mortes?
poisson.pmf(12, 10)

File: dados/estatistica3/poisson.r
# probabilidade exata, por exemplo "2 no segundo dia"
# dpois(x, lambda)

# probabilidade menos que, ou mais que (lower.tail=F)
# ppois(x, lambda, lower.tail=F)

#####################################################

# média de 2 acidentes
# qual a probabilidade de ocorrer 3 ?

dpois(3, lambda=2)

# e a probabilidade de serem 3 ou menos acidentes ?

ppois(3, lambda=2)

# e mais de 3 acidentes ?

ppois(3, lambda = 2, lower.tail = F)
